test/index.test.ts

```diff
@@ -1,92 +1,104 @@
-import nock from 'nock'
-import approvemanApp from '../src'
-import { Probot } from 'probot'
-import prOpenedPayload from './fixtures/basic/pr.opened.json'
-import prReopenedPayload from './fixtures/basic/pr.reopened.json'
-import prSynchronizePayload from './fixtures/basic/pr.synchronize.json'
-import { setConfigToBasic, setConfigNotFound } from './utils/config'
-import { checkApproved, setSinglePreviousReview, verifyReviewDismissed } from './utils/review'
-const fs = require('fs')
-const path = require('path')
+import nock from "nock";
+import approvemanApp from "../src";
+import { Probot } from "probot";
+import prGeneric from "./fixtures/basic/pr.json";
+import { setConfigToBasic, setConfigNotFound } from "./utils/config";
+import { checkSuccessStatus } from "./utils/status";
+import {
+  checkApproved,
+  setSinglePreviousReview,
+  verifyReviewDismissed,
+} from "./utils/review";
+const fs = require("fs");
+const path = require("path");
 
-jest.setTimeout(30000)
+jest.setTimeout(30000);
 
-describe('Approveman tests', () => {
-  let probot: any
-  let mockCert: string
+const prOpenedPayload: any = prGeneric;
```

Comment:
Codacy found an issue: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)